### ISSUE TITLE
Error creating images

### ISSUE THREAD

**Issue by mcquin** [CellProfiler/CellProfiler#2833]
```
==================================================================================== FAILURES =====================================================================================
_________________________________________________________________________ TestLoadData.test_10_01_scaling _________________________________________________________________________

self = <tests.modules.test_loaddata.TestLoadData testMethod=test_10_01_scaling>

    def test_10_01_scaling(self):
        '''Test loading an image scaled and unscaled'''
        folder = "loaddata"
        file_name = "1-162hrh2ax2.tif"
>       path = make_12_bit_image(folder, file_name, (22, 18))

tests/modules/test_loaddata.py:748:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

folder = 'loaddata', filename = '1-162hrh2ax2.tif', shape = (22, 18)

    def make_12_bit_image(folder, filename, shape):
        '''Create a 12-bit image of the desired shape

        folder - subfolder of example images directory
        filename - filename for image file
        shape - 2-tuple or 3-tuple of the dimensions of the image. The axis order
                is i, j, c or y, x, c
        '''
        r = np.random.RandomState()
        r.seed(np.frombuffer(
                hashlib.sha1("/".join([folder, filename])).digest(),
                np.uint8))
        img = (r.uniform(size=shape) * 4095).astype(np.uint16)
        path = os.path.join(example_images_directory(), folder, filename)
        if not os.path.isdir(os.path.dirname(path)):
            os.makedirs(os.path.dirname(path))

        write_image(path, img, PT_UINT16)
        #
        # Now go through the file and find the TIF bits per sample IFD (#258) and
        # change it from 16 to 12.
        #
        with open(path, "rb") as fd:
            data = np.frombuffer(fd.read(), np.uint8).copy()
        offset = np.frombuffer(data[4:8].data, np.uint32)[0]
>       nentries = np.frombuffer(data[offset:offset + 2], np.uint16)[0]
E       IndexError: index 0 is out of bounds for axis 0 with size 0
```

**Comment by mcquin** [comment]
I suspect this error occurred when upgrading prokaryote from `1.0.11` to `2.1.1`.

This may be due to upgrading OME's `formats-*` dependencies without upgrading python-bioformat's loci_tools.jar to a compatible version. I will check if this is indeed the cause.

**Comment by mcquin** [comment]
Syncing the versions didn't fix the issue (but is still a good thing to do).

**Comment by mcquin** [comment]
Resolved #2838
